plaidml/bridge/openvino/plaidml_builder.cpp

```diff
@@ -238,4 +281,11 @@ plaidml::Program buildProgram(const std::shared_ptr<const ngraph::Function>& fun
   return ProgramBuilder(func, netName, inputsInfo, outputsInfo).build();
 }
 
+plaidml::Program buildNodeProgram(const std::shared_ptr<ngraph::Node>& node) {
```

Comment:
Since the resulting plaidml::Program isn't different (just what inputs are used to build it, i.e. Node vs. Function etc), in my opinion it would be clearer to just overload the name buildProgram for this function.

Reply:
I actually like having a separate name, overloading is sometimes confusing.

Reply:
Fair enough, works for me